{
  "meta": {
    "req_path": "./data/ENCO-REs.csv",
    "test_path": "./data/ENCO-STs.csv",
    "mapping_path": "./data/ENCO_mapping.csv",
    "fingerprint": "fp_3b956da36b -> None\nFingerprint changed, expect changes.",
    "input_tokens": 110954,
    "output_tokens": 104
  },
  "data": {
    "links": {
      "S5": [],
      "S63": [],
      "S69": [],
      "S117": [],
      "S125": [],
      "S217": [],
      "S230": [],
      "S232": [],
      "S247": [],
      "B113": [],
      "S184": [],
      "S81": [],
      "B40": [],
      "S95": [
        "147",
        "148"
      ],
      "B47": [],
      "S202": [],
      "S163": [],
      "B89": [],
      "S130": [
        "210"
      ],
      "S131": [],
      "S539": [],
      "S540": [],
      "S226": [],
      "B107": [],
      "S207": [],
      "S220": [],
      "S50": [],
      "S93": [
        "145"
      ],
      "S102": [],
      "S115": [
        "192"
      ],
      "S132": [],
      "S556": [
        "97"
      ],
      "B73": [
        "224"
      ],
      "B87": [],
      "B35": [
        "332"
      ],
      "S13": [],
      "B7": [
        "371"
      ],
      "B102": [
        "378"
      ],
      "S82": [
        "428"
      ],
      "S262": [
        "652"
      ]
    },
    "err": {
      "S63": [
        "Traceback (most recent call last):\n  File \"/cephyr/NOBACKUP/groups/llm-readability/rest-at/LLM-Req-Traceability-alt/src/core/rest.py\", line 279, in to_gpt\n    links = json.loads(curr_res)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/lib/python3.11/json/__init__.py\", line 346, in loads\n    return _default_decoder.decode(s)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/lib/python3.11/json/decoder.py\", line 340, in decode\n    raise JSONDecodeError(\"Extra data\", s, end)\njson.decoder.JSONDecodeError: Extra data: line 2 column 1 (char 3)\n",
        "[]\n[\"T-7\"]"
      ],
      "S125": [
        "Traceback (most recent call last):\n  File \"/cephyr/NOBACKUP/groups/llm-readability/rest-at/LLM-Req-Traceability-alt/src/core/rest.py\", line 279, in to_gpt\n    links = json.loads(curr_res)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/lib/python3.11/json/__init__.py\", line 346, in loads\n    return _default_decoder.decode(s)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/lib/python3.11/json/decoder.py\", line 340, in decode\n    raise JSONDecodeError(\"Extra data\", s, end)\njson.decoder.JSONDecodeError: Extra data: line 2 column 1 (char 3)\n",
        "[]\n[\"T-6\"]"
      ],
      "S132": [
        "Traceback (most recent call last):\n  File \"/cephyr/NOBACKUP/groups/llm-readability/rest-at/LLM-Req-Traceability-alt/src/core/rest.py\", line 279, in to_gpt\n    links = json.loads(curr_res)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/lib/python3.11/json/__init__.py\", line 346, in loads\n    return _default_decoder.decode(s)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/lib/python3.11/json/decoder.py\", line 340, in decode\n    raise JSONDecodeError(\"Extra data\", s, end)\njson.decoder.JSONDecodeError: Extra data: line 2 column 1 (char 3)\n",
        "[]\n[\"T-32\"]"
      ]
    },
    "input_tokens": 110954,
    "output_tokens": 104,
    "system_fingerprint": "fp_3b956da36b -> None\nFingerprint changed, expect changes."
  }
}